package.json

```diff
@@ -1,20 +1,42 @@
 {
   "name": "pippin-ah-frontend",
   "version": "1.0.0",
-  "description": "",
+  "description": "Learngroud React",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "webpack-dev-server --mode development --open --hot",
+    "build": "webpack --mode production"
```

Comment:
We can also have separate build commands for the dev and prod environment.

Reply:
ok, doing that right away